f600233d1b154d507b782f84604bcb12/update/sys_ws_operation_38812576dbb86dd8b1227ea5f3961959.xml

```diff
@@ -40,13 +40,26 @@
 		// validateXML will return null if it passes validation
 		var xmlValid = GlideXMLUtil.validateXML(pdsResponse, false, false);
 		if (xmlValid == null){							
-
 			// Create Portfolio
 			var portfolio = new GlideRecord("x_g_dis_atat_portfolio");
 			portfolio.initialize();
 			portfolio.setValue("name", requestBody.portfolioName);
 			portfolio.setValue("portfolio_status", "PROCESSING");
 			portfolio.setValue("agency", requestBody.portfolioAgency);
+			
+			// set CSP column with first (?) environment name from POST body
```

Comment:
This comment doesn't seem to match the implementation.  My suggestion may be too much.

```suggestion
			// Use the CSP name found in first environment from the POST body to find the sys_id for the desired row in the CSP table .  That row leads to a specific provisioning endpoint in AWS Secrets Manager to be used for the provisioning request.
```